src/main/jbake/content/docs/user/quick-start.adoc

```diff
@@ -40,6 +40,35 @@ bin/standalone.sh  # use standalone.bat on Windows
 You can then navigate your browser to http://localhost:8080/. When you see the login screen,
 you have to register as a user and will then be redirected to the main Hawkular page.
 
+Should you decide to use another hostname and/or port, you'd need to change two files to point to the currect
+URL for Keycloak: `keycloak.json` and `standalone.xml`. Replace the instances of `http://localhost:8080` on the
+property `auth-server-url` by the desired hostname:port
+
+`$HAWKULAR_HOME/modules/org/hawkular/nest/main/deployments/hawkular-console.war/keycloak.json`:
```

Comment:
I think if you put a . at the start of the line, then the file name is used as title box for the listing